octobot_trading/exchanges/parser/util.py

```diff
@@ -0,0 +1,424 @@
+import datetime
+import decimal
+from octobot_commons import timestamp_util
+from octobot_trading import constants as trading_constants
+from octobot_trading.enums import (
+    ExchangePositionCCXTColumns,
+)
+
+
+class Parser:
```

Comment:
I think the file name should be "parser.py"

Reply:
oh yeah its changed